data_structures/linked_list/skip_list.py

```diff
@@ -3,25 +3,56 @@
 https://epaperpress.com/sortsearch/download/skiplist.pdf
 """
 
-from typing import List, Tuple, Optional
+from typing import List, Tuple, Optional, TypeVar, Generic
 from random import random
 
+KT = TypeVar("KT")
```

Comment:
Variable naming can be improved here

Reply:
Naming is same as in [cpython standard library](https://github.com/python/cpython/blob/702e09fd0ad72b248b5adfa0fcfdb58600be77f6/Lib/typing.py#L1533).

Reply:
Will __from typing import KT__ work?

Reply:
Python developers clearly does not want [to export them](https://github.com/python/cpython/blob/702e09fd0ad72b248b5adfa0fcfdb58600be77f6/Lib/typing.py#L1531).